### PR TITLE
Add aiomysql instrumentation

### LINKED ISSUE #754: [Feature] Add instrumentation for aiomysql
Ref #740 

https://github.com/aio-libs/aiomysql

Another bridge driver, it depends on `PyMySQL` so I'm not sure if we need additional instrumentation or not.

### PR DESCRIPTION
**Pull request by Sparkycz**
## Related issues
closes #754

### CODE REVIEW

File: elasticapm/instrumentation/packages/asyncio/aiomysql.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    async def call(self, module, method, wrapped, instance, args, kwargs):
+        if method == "Cursor.execute":
+            query = args[0]
+            name = extract_signature(query)
+
+            # Truncate sql_string to 10000 characters to prevent large queries from
+            # causing an error to APM server.
+            query = shorten(query, string_length=10000)
+
```
Comment (beniwohli) [inline elasticapm/instrumentation/packages/asyncio/aiomysql.py:51]: For the MySQL database to appear in the service map, we need to provide some additional information in the `context.destination` field, like e.g. in the asyncpg instrumentation:

https://github.com/elastic/apm-agent-python/blob/2edeed2f8dd0c59e9a57ddd344a46d7436ff4852/elasticapm/instrumentation/packages/asyncio/asyncpg.py#L60-L65

Would it be possible to add that?

**Review by beniwohli — APPROVED** [review]
This looks great, thanks! If you've got the time, could you add the `context.destination` data? If not, I'll add it before merging :)

File: elasticapm/instrumentation/packages/asyncio/aiomysql.py (hunk 1 above)
Comment (Sparkycz) [reply elasticapm/instrumentation/packages/asyncio/aiomysql.py:51]: Done ;-)

**Comment by Sparkycz** [comment]
I rebased the branch by the current master due to fix conflicts in register.py etc.

**Pull request merged**